provider/storage_benchmark.py

```diff
@@ -335,7 +335,10 @@ def run(self, cmd_options='-a', timeout=1800):
 
 class FioLinuxCfg(object):
     def __init__(self, params, session):
-        fio_pkg = params.get("fio_pkg", 'fio-3.13-48-ga819.tar.bz2')
+        if params.get('os_variant') == 'rhel9':
+            fio_pkg = params.get("fio_pkg", 'fio-3.25.tar.bz2')
+        else:
+            fio_pkg = params.get("fio_pkg", 'fio-3.13-48-ga819.tar.bz2')
```

Comment:
I suggest that you could specify the version in the case level cfg, adding a filter, like `tp-qemu/qemu/tests/cfg/fio_linux.cfg`
```
- fio_linux:
    type = fio_linux
    only Linux
    ...
    fio_options += '${fio_default_options} --rw=randrw'
    RHEL.9:
        fio_pkg = fio-3.13-48-ga819.tar.bz2
```
in case that `fio-3.13-48-ga819 ` does not work later, you just upload another latest version, not need to update framework codes.
Thank.

Reply:
There are many cases invovle fio package, like as throttle test. 
It will bring too many modification in the case level. 
If we have chance to udpate fio in new guest that is good.

Reply:
Yes, you are right, but if just update fio in the guest is not the stable way, like I said, such as network issue, lack of dependency package when updating or installing fio, so still update codes.
Or there is another better way is to add filter in the internal cfg `kar/internal_cfg/guest-os/Linux.cfg`, like:
    ...
    migration_test_command = uname -r
    grub_file = /boot/grub/grub.conf
    RHEL.9:
        fio_pkg = fio-3.13-48-ga819.tar.bz2
    ...

So you could not update the case level, this way is I wanna suggest @zixi-chen yesterday.
Thanks.

Reply:
Thank you all, checked this way, it works, so just update fio package in this patch.

Reply:
> Thank you all, checked this way, it works, so just update fio package in this patch.

Suggest updating the commit log according to the latest codes:
`
deps: Add fio package for RHEL9
`
And we don't need to modify the `storage_benchmark` file, instead implement it in `kar/internal_cfg/guest-os/Linux.cfg` for RHEL9.
Whether I understand your point?
Thanks.

Reply:
yes, internel patch is under-review, too. Change commit log tp update fio package.